libs/loopapp/src/lib/config/testData.ts

```diff
@@ -43,6 +43,799 @@ export const testData: Loop[] = [
     tagStatus: 'RESERVED',
     loopContentStatus: 'OS',
     loopContentProgress: 0.5,
+    woActualCompletionDate: "2023-05-19"

```

Comment:
You can discard this change. Just keep the old testData. I think you need to add woActualCompletionDate property to the test data so it's up-to-date with the new Loop  type